### PR TITLE
ACMS-4124: Updated README.md file.

### PR DESCRIPTION
**Pull request by vishalkhode1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,10 +63,14 @@ Install Acquia CMS using starterkit follow
 
 ## Other Drupal versions
 
-Drupal 10 is installed by default. To install Drupal 9, use the below command:
+Drupal 11 is installed by default. To install Drupal 9, use the below command:
```
Comment (rajeshreeputra) [inline README.md:66]:
```suggestion
Drupal 11 is installed by default. To install Drupal 10, use the below command:
```

File: README.md [hunk 2]
```diff
@@ -63,10 +63,14 @@ Install Acquia CMS using starterkit follow
...
 ```
 composer create-project acquia/drupal-recommended-project:^1
 ```
+To install Drupal 10, use the below command:
+```
+composer create-project acquia/drupal-recommended-project:^2
+```
```
Comment (rajeshreeputra) [inline README.md:73]:
```suggestion
To install Drupal 9, use the below command:
```
composer create-project acquia/drupal-recommended-project:^1
```
```

File: README.md [hunk 3]
```diff
@@ -63,10 +63,14 @@ Install Acquia CMS using starterkit follow
 
 ## Other Drupal versions
 
-Drupal 10 is installed by default. To install Drupal 9, use the below command:
+Drupal 11 is installed by default. To install Drupal 9, use the below command:
 ```
 composer create-project acquia/drupal-recommended-project:^1
```
Comment (rajeshreeputra) [inline README.md:68]:
```suggestion
composer create-project acquia/drupal-recommended-project:^2
```

**Review by rajeshreeputra — CHANGES_REQUESTED** [review]
Looks good please take a look at minor suggestion.

**Review by chandan-singh7929 — COMMENTED** [review]
Running command :
`composer create-project acquia/drupal-recommended-project d11`
Install setup project with core 9.5, please check.

**Review by rajeshreeputra — APPROVED** [review]

**Review by chandan-singh7929 — APPROVED** [review]

**Pull request merged**